code/modules/bitrunning/components/npc_friendly.dm

```diff
@@ -0,0 +1,42 @@
+/// Makes a mob friendly with most NPC factions
+/datum/element/npc_friendly
+
+/datum/element/npc_friendly/Attach(datum/target)
+	. = ..()
+
+	if(!isliving(target))
+		return ELEMENT_INCOMPATIBLE
+
+	var/mob/living/player = target
+
+	player.faction |= list(
+		FACTION_BOSS,
+		FACTION_HIVEBOT,
+		FACTION_HOSTILE,
+		FACTION_MIMIC,
+		FACTION_PIRATE,
+		FACTION_SPIDER,
+		FACTION_STICKMAN,
+		ROLE_ALIEN,
+		ROLE_GLITCH,
+		ROLE_SYNDICATE,
+	)
+
+/datum/element/npc_friendly/Detach(datum/source)
+	. = ..()
+
+	var/mob/living/player = source
+
+	player.faction.Remove(
```

Comment:
couldn't this accidentally remove faction affiliation? faction system kinda sucks tbh are you sure we want a footgun like this?

Reply:
We should at least be storing what factions were on it before and setting it back to that, though that will still cause issues